src/main/java/com/dev/movietheatre/model/dto/UserRegistrationDto.java

```diff
@@ -0,0 +1,43 @@
+package com.dev.movietheatre.model.dto;
+
+import com.dev.movietheatre.annotation.EmailConstraint;
+import com.dev.movietheatre.annotation.PasswordConstraint;
+import javax.validation.constraints.Size;
+
+@PasswordConstraint(
+        password = "password",
+        repeatPassword = "repeatPassword",
+        message = "Passwords do not match!"
+)
+
+public class UserRegistrationDto {
+    @EmailConstraint
+    private String email;
+    @Size(min = 3)
```

Comment:
should we also add this annotation to repeatPassword?

Reply:
Actually yes, to prevent checking process in cause of too short repeatPassword.